script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -1804,16 +1811,17 @@ Yeah... I suppose that could have happened, someplace, somewhere.
 -- Page 111, Offset 41713. Glued.
 -- でも、違うんだ。
 But it didn't.
+But it didn't happen here.
 }
 [sha:b64221a40a1c0e78e3caf370fd355fdda74d8dce]{
 -- Page 111, Offset 41714.
 -- 先輩。あなたのお父さんが、生きている筈は、なくて。
-Senpai, your father couldn't possibly be alive.
+Senpai, your father can't possibly be alive.
 }
 [sha:ca3347dac5f7a8049995e495cf50fbc0972585e5]{
 -- Page 111, Offset 41715.
 -- 俺が、こんな風に、生きている筈は、なくて。
-Just as I couldn't possibly be alive right now either.
+Just as I can't possibly be alive right now either.
```

Comment:
Honestly, on second thought, I'm not a huge fan of these changes, but I can't really tell why